packages/paste-website/src/components/tokens-list/token-card/index.tsx

```diff
@@ -19,7 +19,7 @@ const getTokenAltValue = ({category, value}: {category: string; value: Token['va
     case 'background-colors':
     case 'border-colors':
     case 'text-colors':
-      return rgbToHex(value);
+      return typeof value === 'string' ? rgbToHex(value) : null;
```

Comment:
Question: in what situations is value not a string? I saw that you also had to add `value.toString()` below. What changed?

Reply:
This change was from one of @SiTaggart commits to this branch

Reply:
@shleewhite the type for the token value is either string or number, so we've gotta do some extra protections for that case. @SiTaggart can probably provide additional context if needed.